Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -1018,14 +1004,14 @@ - (void)setWaitingForIndex:(BOOL)waiting {
     if (tagFilterType == SPTagFilterTypeDeleted && waiting)
         return;
     
-    if (waiting && self.navigationItem.titleView != activityIndicator && (self.fetchedResultsController.fetchedObjects.count == 0 || _firstLaunch)){
+    if (waiting && self.navigationItem.titleView != _activityIndicator && (self.fetchedResultsController.fetchedObjects.count == 0 || _firstLaunch)){
         
-        if (!activityIndicator)
-            activityIndicator = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:(SPUserInterface.isDark ? UIActivityIndicatorViewStyleWhite : UIActivityIndicatorViewStyleGray)];
+        if (!_activityIndicator)
```

Comment:
You probably don't want to change much on this PR, but why not move this to the getter of the property in order to create it automatically when it's nil?

Reply:
Sold!! Thank you!